bin/hdfcoinc/pycbc_coinc_hdfinjfind

```diff
@@ -166,22 +177,37 @@ for trigger_file, injection_file in zip(args.trigger_files,
                  (len(found_within_time), len(missed_within_time)))
 
     logging.info('Removing injections in vetoed time')
+
+    # Put individual detector vetoes into list of all vetoed indices, if it
+    # is vetoed in more than one detector, then it will be repeated
```

Comment:
-> 'if an injection is vetoed in more than one ifo its index will appear more than once'